### PR TITLE
[SHELL32] Fix "Max size" Edit box position in Recycle Bin properties page

### PR DESCRIPTION
**Pull request by KyleKatarn31**
## Purpose

_ Edit box is wrongly placed

image

JIRA issue: [CORE-16972](https://jira.reactos.org/browse/CORE-16972)

## Proposed changes

_ Redefinition of edit box position

image

### CODE REVIEW

File: dll/win32/shell32/lang/it-IT.rc [hunk 1]
```diff
@@ -384,7 +384,7 @@ BEGIN
     CONTROL "", 14000, "SysListView32", LVS_REPORT | LVS_SHOWSELALWAYS | LVS_SHAREIMAGELISTS | WS_BORDER | WS_TABSTOP, 10, 10, 220, 50
     GROUPBOX "Impostazioni per la posizione indicata", -1, 10, 72, 220, 70
     RADIOBUTTON "&Dimensione personalizzata:", 14001, 20, 90, 80, 10, WS_TABSTOP
-    EDITTEXT 14002, 106, 87, 50, 14, WS_TABSTOP | ES_NUMBER
+    EDITTEXT 14002, 116, 103, 50, 14, WS_TABSTOP | ES_NUMBER
     LTEXT "Dimensione &massima(MB):", -1, 20, 105, 70, 10
```
Comment (binarymaster) [inline dll/win32/shell32/lang/it-IT.rc:388]:
```suggestion
    LTEXT "Dimensione &massima(MB):", -1, 20, 105, 86, 10
```

File: dll/win32/shell32/lang/it-IT.rc [hunk 2]
```diff
@@ -384,7 +384,7 @@ BEGIN
     CONTROL "", 14000, "SysListView32", LVS_REPORT | LVS_SHOWSELALWAYS | LVS_SHAREIMAGELISTS | WS_BORDER | WS_TABSTOP, 10, 10, 220, 50
     GROUPBOX "Impostazioni per la posizione indicata", -1, 10, 72, 220, 70
     RADIOBUTTON "&Dimensione personalizzata:", 14001, 20, 90, 80, 10, WS_TABSTOP
```
Comment (binarymaster) [inline dll/win32/shell32/lang/it-IT.rc:386]:
```suggestion
    RADIOBUTTON "&Dimensione personalizzata:", 14001, 20, 90, 101, 10, WS_TABSTOP
```

File: dll/win32/shell32/lang/pt-PT.rc [hunk 3]
```diff
@@ -384,7 +384,7 @@ BEGIN
     CONTROL "", 14000, "SysListView32", LVS_REPORT | LVS_SHOWSELALWAYS | LVS_SHAREIMAGELISTS | WS_BORDER | WS_TABSTOP, 10, 10, 220, 50
     GROUPBOX "Propriedades para as localizações seleccionadas", -1, 10, 72, 220, 70
     RADIOBUTTON "&Tamanho personalizado:", 14001, 20, 90, 80, 10, WS_TABSTOP
```
Comment (binarymaster) [inline dll/win32/shell32/lang/pt-PT.rc:386]:
```suggestion
    RADIOBUTTON "&Tamanho personalizado:", 14001, 20, 90, 92, 10, WS_TABSTOP
```

File: dll/win32/shell32/lang/pt-PT.rc [hunk 4]
```diff
@@ -384,7 +384,7 @@ BEGIN
...
-    EDITTEXT 14002, 106, 87, 50, 14, WS_TABSTOP | ES_NUMBER
+    EDITTEXT 14002, 116, 103, 50, 14, WS_TABSTOP | ES_NUMBER
     LTEXT "Tamanho M&áximo(MB):", -1, 20, 105, 70, 10
```
Comment (binarymaster) [inline dll/win32/shell32/lang/pt-PT.rc:388]:
```suggestion
    LTEXT "Tamanho M&áximo(MB):", -1, 20, 105, 75, 10
```

File: dll/win32/shell32/lang/pt-BR.rc [hunk 5]
```diff
@@ -384,7 +384,7 @@ BEGIN
     CONTROL "", 14000, "SysListView32", LVS_REPORT | LVS_SHOWSELALWAYS | LVS_SHAREIMAGELISTS | WS_BORDER | WS_TABSTOP, 10, 10, 220, 50
     GROUPBOX "Configurações para local selecionado", -1, 10, 72, 220, 70
     RADIOBUTTON "&Tamanho personalizado:", 14001, 20, 90, 80, 10, WS_TABSTOP
```
Comment (binarymaster) [inline dll/win32/shell32/lang/pt-BR.rc:386]:
```suggestion
    RADIOBUTTON "&Tamanho personalizado:", 14001, 20, 90, 92, 10, WS_TABSTOP
```

File: dll/win32/shell32/lang/pt-BR.rc [hunk 6]
```diff
@@ -384,7 +384,7 @@ BEGIN
...
-    EDITTEXT 14002, 106, 87, 50, 14, WS_TABSTOP | ES_NUMBER
+    EDITTEXT 14002, 116, 103, 50, 14, WS_TABSTOP | ES_NUMBER
     LTEXT "Tamanho &Máximo (MB):", -1, 20, 105, 70, 10
```
Comment (binarymaster) [inline dll/win32/shell32/lang/pt-BR.rc:388]:
```suggestion
    LTEXT "Tamanho &Máximo (MB):", -1, 20, 105, 75, 10
```

File: dll/win32/shell32/lang/sk-SK.rc [hunk 7]
```diff
@@ -384,7 +384,7 @@ BEGIN
     CONTROL "", 14000, "SysListView32", LVS_REPORT | LVS_SHOWSELALWAYS | LVS_SHAREIMAGELISTS | WS_BORDER | WS_TABSTOP, 10, 10, 220, 50
     GROUPBOX "Nastavenia pre zvolené umiestnenie", -1, 10, 72, 220, 70
     RADIOBUTTON "&Vlastná veľkosť:", 14001, 20, 90, 80, 10, WS_TABSTOP
-    EDITTEXT 14002, 106, 87, 50, 14, WS_TABSTOP | ES_NUMBER
+    EDITTEXT 14002, 116, 103, 50, 14, WS_TABSTOP | ES_NUMBER
     LTEXT "&Maximálna veľkosť (MB):", -1, 20, 105, 70, 10
```
Comment (binarymaster) [inline dll/win32/shell32/lang/sk-SK.rc:388]:
```suggestion
    LTEXT "&Maximálna veľkosť (MB):", -1, 20, 105, 80, 10
```

File: dll/win32/shell32/lang/id-ID.rc [hunk 8]
```diff
@@ -384,7 +384,7 @@ BEGIN
     CONTROL "", 14000, "SysListView32", LVS_REPORT | LVS_SHOWSELALWAYS | LVS_SHAREIMAGELISTS | WS_BORDER | WS_TABSTOP, 10, 10, 220, 50
     GROUPBOX "Pengaturan untuk lokasi yang dipilih", -1, 10, 72, 220, 70
     RADIOBUTTON "&Ukuran khusus:", 14001, 20, 90, 80, 10, WS_TABSTOP
-    EDITTEXT 14002, 106, 87, 50, 14, WS_TABSTOP | ES_NUMBER
+    EDITTEXT 14002, 116, 103, 50, 14, WS_TABSTOP | ES_NUMBER
     LTEXT "Ukur&an maksimal (MB):", -1, 20, 105, 70, 10
```
Comment (binarymaster) [inline dll/win32/shell32/lang/id-ID.rc:388]:
```suggestion
    LTEXT "Ukur&an maksimal (MB):", -1, 20, 105, 74, 10
```

**Comment by binarymaster** [comment]
@HBelusca why rush? :confused:

**Comment by HBelusca** [comment]
Please create a separate PR for implementing the fixes suggested by @binarymaster

**Comment by KyleKatarn31** [comment]
Sorry, I only implemented the one that were on the exact same pattern as FR and US. For the ones that were customized, I let the relevant translator fix accordingly.

File: dll/win32/shell32/lang/it-IT.rc (hunk 1 above)
Comment (KyleKatarn31) [reply dll/win32/shell32/lang/it-IT.rc:388]: Dedicated PR please.

File: dll/win32/shell32/lang/pt-PT.rc (hunk 3 above)
Comment (KyleKatarn31) [reply dll/win32/shell32/lang/pt-PT.rc:386]: Dedicated PR please.

File: dll/win32/shell32/lang/pt-PT.rc (hunk 4 above)
Comment (KyleKatarn31) [reply dll/win32/shell32/lang/pt-PT.rc:388]: Dedicated PR please.

File: dll/win32/shell32/lang/pt-BR.rc (hunk 5 above)
Comment (KyleKatarn31) [reply dll/win32/shell32/lang/pt-BR.rc:386]: Dedicated PR please.

File: dll/win32/shell32/lang/pt-BR.rc (hunk 6 above)
Comment (KyleKatarn31) [reply dll/win32/shell32/lang/pt-BR.rc:388]: Dedicated PR please.

File: dll/win32/shell32/lang/sk-SK.rc (hunk 7 above)
Comment (KyleKatarn31) [reply dll/win32/shell32/lang/sk-SK.rc:388]: Dedicated PR please.

File: dll/win32/shell32/lang/it-IT.rc (hunk 2 above)
Comment (KyleKatarn31) [reply dll/win32/shell32/lang/it-IT.rc:386]: Dedicated PR please.

**Pull request merged**